### PR TITLE
Designing runbooks recommendations and best practices

### PR DESCRIPTION
**Pull request by OctopusBob**
# About

This is a post about designing runbooks and what are recommendations and best practices to consider to ensure the runbook is meeting the needs of our customers.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

## How to review this PR
Feels like a standard opinion piece for the blog :)

## Publication date
This is an extension of the webinar being broadcast on October 28th, 2020.  It doesn't have to go out right away.  I mention it in the webinar in the event they want to read instead of watch.

### CODE REVIEW

**Review by robpearson — APPROVED** [review]
Brilliant post Bob! I really enjoyed reading it. Your experience really shines through. :) 

I added a couple comments but nothing major. I do we need the right title so people find this post. It's fantastic. :)

File: blog/2020-10/runbook-recommendations-and-best-practices/index.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+title: Runbook recommendations and best practices
```
[the remark this answers is not in the document]
Comment (OctopusBob) [reply blog/2020-10/runbook-recommendations-and-best-practices/index.md]: I like it a lot.  I've pushed an update along with some line edits to clean up some text.

File: blog/2020-10/runbook-recommendations-and-best-practices/index.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+- Has this been tried before?  If so, what worked and what didn't work?
+- Where should the runbook run?
+- Who will invoke, and how will this runbook be invoked?  
+- Should running the runbook require approval?
+- Should there be a notification?  If so, when should they go out, and what information should be included?  How can I maintain a high signal to noise ratio?
+- Is the runbook project-specific?  Or is it generic?  
+- Is there any information needing to be captured and retained?  What should the retention policy be?  Should I push anything to another service?
+
+:::highlight
+**Recommendation:** Take the time to go through a similar process.  A little prep work goes a long way.
```
[the remark this answers is not in the document]
Comment (OctopusBob) [reply blog/2020-10/runbook-recommendations-and-best-practices/index.md:36]: Denied!

File: blog/2020-11/runbook-recommendations-and-best-practices/index.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
+---
+title: Runbook lessons learned and recommendations
+description: Recommendations from lessons learned on how to design and implement runbooks
+author: bob.walker@octopus.com
+visibility: public
+published: 2099-12-31
```
Comment (wordlee) [inline blog/2020-11/runbook-recommendations-and-best-practices/index.md:6]:
```suggestion
published: 2020-11-18
```

**Pull request merged**